t/14-grutasks-git.t

```diff
@@ -448,12 +449,114 @@ subtest 'delete_needles' => sub {
+    is($outcome, 'SKIP', 'Skips when lock file for gitlab is absent');
+};
+
+subtest 'GitServerOutage.pm: file present but younger than 1800 => skip' => sub {
+    my $tmpdir = tempdir();
+    local $ENV{OPENQA_GIT_SERVER_OUTAGE_FILE} = "$tmpdir/mock";
+    my $mock_file = "$tmpdir/mock.gitlab.lock";
+    path($mock_file)->touch;
+    my $outcome = OpenQA::Git::GitServerOutage::check_availability($t->app, 'gitlab');
+    is($outcome, 'SKIP', 'Skips the job for an mtime < 1800');
+};
+
+subtest 'GitServerOutage.pm: file present and older => fail' => sub {
+    my $tmpdir = tempdir();
+    local $ENV{OPENQA_GIT_SERVER_OUTAGE_FILE} = "$tmpdir/mock";
+    my $mock_file = "$tmpdir/mock.gitlab.lock";
+    path($mock_file)->touch;
+    my $old_time = time - 2000;
+    utime($old_time, $old_time, $mock_file)
+      or die "Couldn't change mtime on file: $mock_file - $!";
+    my $outcome = OpenQA::Git::GitServerOutage::check_availability($t->app, 'gitlab');
+    is($outcome, 'FAIL', 'Fails the job when mtime >= 1800');
+};
+
+subtest 'GitServerOutage.pm: remove_state_file removes existing lock file' => sub {
+    my $tmpdir = tempdir();
+    local $ENV{OPENQA_GIT_SERVER_OUTAGE_FILE} = "$tmpdir/mock";
+    my $mock_file = "$tmpdir/mock.gitlab.lock";
+    path($mock_file)->touch;
+    ok(-f $mock_file, 'mock.gitlab.lock exists initially');
+    OpenQA::Git::GitServerOutage::remove_state_file($t->app, 'gitlab');
+    ok(!-f $mock_file, 'gitlab lock file was removed');
+};
+
+subtest 'GitServerOutage.pm: multiple servers do not conflict' => sub {
+    my $tmpdir = tempdir();
+    local $ENV{OPENQA_GIT_SERVER_OUTAGE_FILE} = "$tmpdir/mock";
+    my $outcome_gitlab = OpenQA::Git::GitServerOutage::check_availability($t->app, 'gitlab');
+    is($outcome_gitlab, 'SKIP', 'No gitlab lock => skip');
+    my $gitlab_file = "$tmpdir/mock.gitlab.lock";
+    ok(-f $gitlab_file, 'Created mock.gitlab.lock');
+    my $outcome_github = OpenQA::Git::GitServerOutage::check_availability($t->app, 'github');
+    is($outcome_github, 'SKIP', 'No github lock => skip');
+    my $github_file = "$tmpdir/mock.github.lock";
+    ok(-f $github_file, 'Created mock.github.lock');
+
+    my $old_time = time - 2000;
+    utime($old_time, $old_time, $github_file);
+    # Re-check
+    $outcome_gitlab = OpenQA::Git::GitServerOutage::check_availability($t->app, 'gitlab');
+    $outcome_github = OpenQA::Git::GitServerOutage::check_availability($t->app, 'github');
+    is($outcome_gitlab, 'SKIP', 'GitLab file still fresh => skip');
+    is($outcome_github, 'FAIL', 'GitHub file older => fail');
+};
+
+subtest 'Clone.pm: Internal API unreachable => skip' => sub {
+    my $tmpdir = tempdir();
+    local $ENV{OPENQA_GIT_SERVER_OUTAGE_FILE} = "$tmpdir/mock.other.lock";
+    my $clone_mock = Test::MockModule->new('OpenQA::Task::Git::Clone');
+    $clone_mock->redefine(
+        _git_clone => sub {
+            die "Internal API unreachable\n";
+        });
```

Comment:
```suggestion
    $clone_mock->redefine(_git_clone => sub { die "Internal API unreachable\n" });
```
The same counts for the other occurrence and a few other cases where things probably just fit in one line.